### PR TITLE
Veeam backup SNMP Traps

### PR DESCRIPTION
**Pull request by mpikzink**
Log Veeam backup status in eventlog

Please give a short description what your pull request is for

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: tests/Feature/SnmpTraps/VeeamTrapTest.php [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+VEEAM-MIB::backupJobName SureBackup Job
+VEEAM-MIB::sourceHostName hostname
+VEEAM-MIB::vmBackupComment 
+VEEAM-MIB::vmBackupResult Success";
+
+        $trap = new Trap($trapText);
+
+        $message = 'SNMP Trap: Backup success - SureBackup Job 1';
+        Log::shouldReceive('event')->once()->with($message, $device->device_id, 'backup', 2);
+
```
Comment (murrant) [inline tests/Feature/SnmpTraps/VeeamTrapTest.php:30]: Looks like you are getting an extra log entry the tests don't expect or your log message doesn't match.

Comment (mpikzink) [reply tests/Feature/SnmpTraps/VeeamTrapTest.php:30]: mmh, I do not find my issue for a view days. No idea how to debug this test cases :-( Can someone help me?

**Comment by murrant** [comment]
@mpikzink it helps to capture actual traps for the tests.   The ones you added seem to be fabricated, because there was several incorrect things in the test trap strings.

**Comment by mpikzink** [comment]
THX @murrant I will do in my next PRs. I have 2 more trap handlers in the pipeline. Is there a good way to capture? I grabbed it from the devices event log with debugging on and reformatted it. There I did the mistakes.

**Review by murrant — APPROVED** [review]

**Comment by murrant** [comment]
How I capture is in snmptrap.php, I write the $text to a file.

**Pull request merged**